src/components/inputs/FormImageUploader.tsx

```diff
@@ -15,27 +15,30 @@ enum ByteUnit {
 // TODO: This is a double up of `ImageUploader`. We should combine the two.
 
 export const FormImageUploader = ({
-  value,
```

Comment:
Unused code

Reply:
Great pickup @wraeth-eth. I've reinstated `value` and incorporated it with my changes as necessary. The issue you attached is fixed:

https://user-images.githubusercontent.com/96150256/187572017-595b4ba8-4dcf-4891-84e9-c7aa693b5e45.mp4